src/ui/Form/Checkbox.tsx

```diff
@@ -53,11 +50,16 @@ export default function Checkbox<FormValues extends FieldValues>({
       </Form.CheckboxLabel>
 
       <Form.TextWrapper>
-        <Form.ErrorMessage id={id + 'error'}>
-          {fieldState.error?.message}
-        </Form.ErrorMessage>
-        <Form.Description>{inputDescription}</Form.Description>
+        {fieldState.invalid ? (
```

Comment:
Even when there's an error, helper text should still be visible.

Reply:
This is not the behavior we're having the whole time and also not what Airbnb has

![image](https://user-images.githubusercontent.com/54929186/140526649-9a4f2ad4-bd11-4833-96cb-3278e8968eaf.png)

![image](https://user-images.githubusercontent.com/54929186/140526710-b9af806c-6333-405a-b25e-fc4dea1665c4.png)

I actaully like this better